api/services/locations.js

```diff
@@ -0,0 +1,170 @@
+import config from 'config';
+import AWS from 'aws-sdk';
+import { has, sortBy } from 'lodash';
+import { db } from './db';
+
+// Athena config
+const {
+  accessKeyId,
+  secretAccessKey,
+  OutputLocation,
+  Database,
+  region,
+  measurementsTable
+} = config.get('athena');
+
+// Athena client
+const athena = new AWS.Athena({
```

Comment:
Is there anyway to reuse the code at https://github.com/openaq/openaq-api/blob/develop/api/services/athena.js or `little-owl` instead of rewriting all new logic here?